lasp_opensearch_data_center/constructs/frontend.py

```diff
@@ -164,6 +170,9 @@ def __init__(
             ],
         )
 
+        # Currently the /frontend object path in the S3 bucket is restricted to entities with this tag
+        Tags.of(self.frontend_deployment_role).add("group", "frontend")
```

Comment:
Can you just remind me why we did this based on tags? Obviously it works well but why not just directly reference the bucket ARN?

Reply:
This was a pattern we had used in SpaceWeather to restrict specific folders to specific groups.

I'll remove this tag, as we already have it locked down to the frontend/ object path in lines 163-167:

                    resources=[
                        # All bucket objects must start with the keyword "frontend"
                        # Additional restrictions imposed at the resource/bucket level
                        frontend_bucket.bucket_arn,
                        frontend_bucket.bucket_arn + "/frontend/*",


Any other updates you'd like here?

Reply:
As part of that I'd remove this entire block from the frontend_storage construct as well:

 # Restrict puts/deletes to the "frontend/" folder to a specific tag
        # An IAM user must be generated for the webteam via the AWS CLI
        # and tagged with group:frontend
        #
        # For the DEV account the Web team will put their built JS apps into this object path:
        # "frontend/libera/live/"
        # For the PROD account the Web team will put their built JS apps into this versioned object path:
        # "frontend/libera/VXX/"
        #
        # All webteam PUTs will be performed via Jenkins using CLI credentials
        self.frontend_bucket.add_to_resource_policy(
            aws_iam.PolicyStatement(
                effect=aws_iam.Effect.DENY,
                principals=[aws_iam.AnyPrincipal()],
                actions=[
                    "s3:DeleteObject",
                    "s3:PutObject",
                ],
                resources=[
                    self.frontend_bucket.bucket_arn,
                    self.frontend_bucket.arn_for_objects("frontend/*"),
                ],
                conditions={"StringNotLike": {"aws:PrincipalTag/group": "*frontend*"}},
            )
        ),

Reply:
Yeah, I think that's a more visible way to manage permissions. It's easy to see which bucket has this permission. Hard to see which bucket has a specific tag.